Makefile

```diff
@@ -42,7 +42,7 @@ GOLANGCI_LINT := $(TOOLS_BIN_DIR)/golangci-lint
 MOCKGEN := $(TOOLS_BIN_DIR)/mockgen
 
 # Define Docker related variables. Releases should modify and double check these vars.
-REGISTRY ?= gcr.io/$(shell gcloud config get-value project)
+REGISTRY ?= k8scloudprovider
```

Comment:
The idea was to implement the regular staging process for cluster api images, for which I'm currently building another PR. I would really like to do it like every other provider and not push to dockerhub instead.

I'll open a PR for that shortly. If somebody wants to push to dockerhub meanly it can always be overwritten by using `export REGISTRY`

Reply:
ok, I don't know the release process
 the reason I add this is because I am tring to build image and can't build it...
and I agree this might be a better way to be same to other providers